.github/workflows/ci.yml

```diff
@@ -51,3 +51,19 @@ jobs:
       - name: Deploy to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@v1
+
+  lint:
```

Comment:
automatically run linter on pull requests

Reply:
That will be nice!